app-admin/mkosi/mkosi-v11.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="A tool for generating OS images"
+HOMEPAGE="https://github.com/systemd/mkosi"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="+debian"
+KEYWORDS=""
+RDEPEND="
+	sys-apps/systemd
+	dev-vcs/git
+	debian? ( dev-util/debootstrap )
+"
+
+distutils_enable_tests pytest
+
+if [[ "${PV}" == "9999" ]]; then
```

Comment:
This is a nit and we can fix it on merge but usually this block goes right under HOMEPAGE.